rpcserver.go

```diff
@@ -2735,6 +2735,26 @@ func (r *rpcServer) DeleteAssetRoot(ctx context.Context,
 
 	rpcsLog.Debugf("Deleting asset root for %v", spew.Sdump(universeID))
 
+	// If the universe proof type is unspecified, we'll delete both the
```

Comment:
Heh forgot we even had this call....

Good to have though as gives the ability to properly prune a tree if needed (delete all transfer and just go back to issuance, etc). 

Don't think we need to do it here, but I think in the future we'd also want to add a DB record if something should be deleted, then just never added back again.